lib/refile/attachment.rb

```diff
@@ -34,8 +34,9 @@ module Attachment
     # @param [Symbol, nil] type                         The type of file that can be uploaded, see {Refile.types}
     # @param [String, Array<String>, nil] extension     Limit the uploaded file to the given extension or list of extensions
     # @param [String, Array<String>, nil] content_type  Limit the uploaded file to the given content type or list of content types
+    # @param [true, false] destroy                      Whether to remove stored file if its model is destroyed
     # @return [void]
-    def attachment(name, cache: :cache, store: :store, raise_errors: true, type: nil, extension: nil, content_type: nil)
+    def attachment(name, cache: :cache, store: :store, raise_errors: true, type: nil, extension: nil, content_type: nil, destroy: true)
```

Comment:
I can change that. I think it's best to be explicit. I'll move the documentation up to `lib/refile/attachment/active_record.rb`. So something like this?

```ruby
def attachment(name, raise_errors: false, destroy: true, **options)
  super(name, raise_errors: false, **options)
```